website/views.py

```diff
@@ -219,8 +222,19 @@ def get_dashboard_nodes(auth):
     return _render_nodes(response_nodes, auth)
 
 
+def homepage_snapshot():
```

Comment:
This is required to make crawler call the homepage(i.e. osf.io). Usually, crawlwer calls are made when a javascript makes api calls to OSF, a corresponding crawler call is made. Since there were no API calls made when homepage was called, I've added this API view function.